tests/cli.js

```diff
@@ -0,0 +1,91 @@
+const tape = require('tape')
+const c = require('./../lib/constants.js')
+
+function requireUncached (module) {
+  delete require.cache[require.resolve(module)]
+  return require(module)
+}
+
+tape('[CLI]: options', t => {
+  t.test('should test defaults', st => {
+    const cliParser = requireUncached('./../lib/cliParser.js')
+    const config = cliParser.getClientConfig()
+
+    st.equal(config.networkid, c.DEFAULT_NETWORK_ID, 'default networkid is set')
+    st.equal(config.loglevel, c.DEFAULT_LOG_LEVEL, 'default loglevel is set')
+    st.equal(config.rpcport, c.DEFAULT_RPC_PORT, 'default rpcport is set')
+    st.equal(config.rpcaddr, c.DEFAULT_RPC_ADDR, 'default rpcaddr is set')
+
+    delete require.cache[require.resolve('yargs')]
+
+    st.end()
```

Comment:
Pretty cool and clean structure, also tested your branch locally, will approve, thanks again for the PR!